src/libraries/System.Net.Http.WinHttpHandler/pkg/System.Net.Http.WinHttpHandler.pkgproj

```diff
@@ -1,19 +1,11 @@
 ﻿<Project DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
   <ItemGroup>
-    <ProjectReference Include="..\ref\System.Net.Http.WinHttpHandler.csproj">
```

Comment:
Please double check the diff on this nuspec to make sure that this didn't remove `Exclude="Compile"` from some dependencies.  If it did, share those and make sure we're OK with such a change.

Reply:
I diffed both the nuspec and the package content. That's why included the change as it didn't result in a different output.

Reply:
Confirmed